README.md

```diff
@@ -85,23 +121,37 @@ npm test
 
 #### Requirements
 
-All the tests are designed to test end to end scenarios, and there are some requirements for its current execution:
-- Mosquitto v1.3.5 server running
-- MongoDB v3.x server running
+All the tests are designed to test end to end scenarios, and there are some
+requirements for its current execution:
 
-#### Execution
+-   Mosquitto v1.3.5 server running
+-   MongoDB v3.x server running
 
-To run tests, type
-```bash
-grunt test
-```
 
-Tests reports can be used together with Jenkins to monitor project quality metrics by means of TAP or XUnit plugins.
-To generate TAP report in `report/test/unit_tests.tap`, type
-```bash
-grunt test-report
-```
+## Quality Assurance
```

Comment:
Same comment to the one published at https://github.com/telefonicaid/iotagent-ul/pull/301#discussion_r236292514

Reply:
> This section should include:
> 
> Information about the last published version (so the user can know the gap between "version tested" and the newest one, evaluating if the information is obsolete enough for him or not)
>
> A link to the labeling calculation procedure, i.e. which metrics and in which form decide which label to assign. Ideally, this should be a mathematical formula/table leaving no room for subjective interpretation.
>
> A link (or several) to the report(s) in which the labels are based, explaining which particular values of the metrics have been used (and how they has been got) in the above formula.
>
> In addition, why two of them are in "no results" status?

All these are good suggestions, but I am unable to implement the requested changes at the moment.

The badges are dynamically driven from the data held in a series of JSON files. The data you are requesting is currently not present within FIWARE. The existing files only hold the **version tested** and not the **current version.** , FIWARE QA don't hold the current version for this component -that is under Telefonica's control

Regarding the testing procedure, the following links within the FIWARE catalogue describe the process:

* https://github.com/Fiware/catalogue#testing
* https://www.fiware.org/developers/qa-labeling-of-fiware-components/

It is not part of the current [contribution requirements](https://github.com/Fiware/contribution-requirements) to display additional links and information - I agree that the QA section could be expanded sometime, but the limited information within this block is currently just a summary of the parts which are available. If you want to display more that is fine, but it would make sense to add a PR to the requirements so that all enablers display the same information consistently.

The two missing ("no results" status) badges are because the QA data has not been updated yet.

Reply:
I think that in general is not a good idea trying to abstract a very complex reality (as the support, documentation or stability of a given component are) with just a plain letter. It may work in the energy efficiency context (which I understand is from where this has been mirrored), but when software comes into play, this is much more complex. An IOTA is not like a refrigerator :)

However, if we want to use a letter for that (I guess that for a marketing purpose), then you have to give the user enough evidence to know how we haved ended with that letter. Including just the label is, on the one hand, unclear for the user (*“ok, I as a developer see A+++ in stability but how can I be sure your component honours it and you developer are not exaggerating?”*) and, on the other hand, can be unfair for the component development team (*“ok, I as a developer see a D in documentation but how I can know which particular documents have been taken into account and which problems have been found?”*)

I understand that FIWARE rules doesn’t oppose to include such detail in this section and that if it cannot be done right now is due to we lack that information in the proper format or place. Thus, is a problem on the maturity of the procedure, which should provide (in the JSON file you mention for instance) such information. In that case, let’s put this on hold while it gets mature.

(I understand this could need further discussion, so in order not blocking this PR what I suggest is to leave this section with a “Under discussion” content as placeholder in this PR, then open a new PR which the section content once this gets merged to continue the dicussion there)